lib/kamal/secrets/adapters/aws_secretsmanager.rb

```diff
@@ -0,0 +1,25 @@
+class Kamal::Secrets::Adapters::AwsSecretsmanager < Kamal::Secrets::Adapters::Base
```

Comment:
The offical name is `AWS Secrets Manager`, so let's call this `AwsSecretsManager` (and rename the file to `aws_secrets_manager.rb`

Reply:
It's funny, I went back and forth on the name but ultimately decided on one word because the AWS CLI calls it "secretsmanager" over "secrets-manager". I agree though, I like the two word version better and will get that changed.